src/test/groovy/org/gradle/android/WorkaroundTest.groovy

```diff
@@ -12,9 +12,9 @@ class WorkaroundTest extends Specification {
         workarounds.collect { it.class.simpleName.replaceAll(/Workaround/, "") }.sort() == expectedWorkarounds.sort()
         where:
         androidVersion  | expectedWorkarounds
-        "7.3.0-alpha07" | ['ZipMergingTask', 'LibraryJniLibs', 'DataBindingMergeDependencyArtifacts', 'BundleLibraryClasses', 'MergeSourceSetFolders', 'RoomSchemaLocation', 'StripDebugSymbols', 'MergeNativeLibs', 'CompileLibraryResources']
-        "7.2.0-beta04" | ['ZipMergingTask', 'LibraryJniLibs', 'DataBindingMergeDependencyArtifacts', 'BundleLibraryClasses', 'MergeSourceSetFolders', 'RoomSchemaLocation', 'StripDebugSymbols', 'MergeNativeLibs', 'CompileLibraryResources']
-        "7.1.3"  | ['ZipMergingTask', 'LibraryJniLibs', 'DataBindingMergeDependencyArtifacts', 'BundleLibraryClasses', 'MergeSourceSetFolders', 'RoomSchemaLocation', 'StripDebugSymbols', 'MergeNativeLibs', 'CompileLibraryResources']
+        "7.3.0-alpha08" | ['CompileLibraryResources', 'MergeNativeLibs', 'MergeSourceSetFolders', 'RoomSchemaLocation', 'ZipMergingTask']
```

Comment:
did you format/alphabetize these? could you do that for the rest of them as well?

Reply:
I've removed some expected tasks for `7.3.0-alpha08` and `7.2.0-beta04` and reformated the whole so `|` are aligned vertically. By "the rest of them" do you mean other similar tests ?

Reply:
no, not the other tests, i mean the lines below since they are not in the same alphabetical order now.

Reply:
👍 , I've alphabetized all of them